Makefile

```diff
@@ -181,7 +181,7 @@ KUSTOMIZE_CONFIG_CERT_MANAGER ?= https://github.com/cert-manager/cert-manager/re
 .PHONY: deploy-cert-manager
 deploy-cert-manager:
 	kubectl apply -f $(KUSTOMIZE_CONFIG_CERT_MANAGER)
-	kubectl -n cert-manager wait --for=condition=Available deployment \
+	kubectl -n cert-manager wait --timeout=2m --for=condition=Available deployment \
```

Comment:
why do we need 2m timeouts?

Reply:
Because sometime 1m isn't enough and cert-manager isn't ready.